service/controller/control_plane_resource_set.go

```diff
@@ -347,6 +363,7 @@ func newControlPlaneResourceSet(config controlPlaneResourceSetConfig) (*controll
 		awsClientResource,
 		accountIDResource,
 		encryptionSearcherResource,
+		tccpOutputsResource,
```

Comment:
We actually need some values in the controllercontext such as hosted zone ID which are only stored in the `tccp` outputs so I'm bringing it back but ordering it before `tcccpn` so that `OperatorVersion` isn't overwritten.